chapel.html.markdown

```diff
@@ -893,10 +954,11 @@ foo();
 // We can declare a main procedure, but all the code above main still gets
 // executed.
 proc main() {
+  writeln("PARALLELISM START");
```

Comment:
Usually we remove this chapel/test artifact from the tutorial code.

Reply:
Yes. I agree.

Reply:
Done.